### PR TITLE
better coverstore imports

### PR DESCRIPTION
**Pull request by hornc**
Small improvement, noticed while working on #2293

### CODE REVIEW

**Comment by cclauss** [comment]
OPTIONAL: We can _force_ Python 2 to behave like Python 3 in this regard by adding
`from __future__ import absolute_import`

**Comment by hornc** [comment]
The **Blocked** status was added to prevent this being merged before the next due Open Library release on (US) Thursday since changes to imports are risky and the testing around coverstore is not giving enough confidence that this change won't break things.

Current plan is to merge first thing next week (20 Jan 2020). In the meantime I'll see if I can find a way to increase confidence in this change.

File: openlibrary/coverstore/code.py [hunk 1]
```diff
@@ -9,9 +9,9 @@
 import array
 import memcache
 
-import db
-import config
-from utils import safeint, rm_f, random_string, ol_things, ol_get, changequery, download
+from openlibrary.coverstore import db
+from openlibrary.coverstore import config
```
Comment (cclauss) [inline openlibrary/coverstore/code.py]: Compress lines 12 and 13 into one line?

File: openlibrary/coverstore/code.py [hunk 2]
```diff
@@ -9,9 +9,9 @@
...
+from openlibrary.coverstore.utils import safeint, rm_f, random_string, ol_things, ol_get, changequery, download
 
 from coverlib import save_image, read_image, read_file
```
Comment (cclauss) [inline openlibrary/coverstore/code.py]: This one also needs to be an absolute import.

**Comment by hornc** [comment]
Closing in favour of #2867

**Pull request closed without merging**